src/MCPClient/lib/clientScripts/elasticSearchIndexProcessAIP.py

```diff
@@ -51,12 +57,20 @@
         uuid = sys.argv[2]
         sipName = sys.argv[3]
 
+        # Even though we treat MODS identifiers as SIP-level, we need to index them here
+        # because the archival storage tab actually searches on the aips/aipfile index.
+        mods_paths = list_mods(pathToAIP)
+        identifiers = []
+        for mods in mods_paths:
+            identifiers.extend(extract_identifiers_from_mods(mods))
+
         exitCode = elasticSearchFunctions.connect_and_index_files(
             'aips',
             'aipfile',
             uuid,
             pathToAIP,
-            sipName
+            sipName,
+            identifiers=identifiers
```

Comment:
```identifiers``` is the last param here, but the second-last in the definition.  sipName needs to be ```sipName=sipName``` or moved to the end.